src/CFamily.UnitTests/CMake/CompilationDatabaseLocatorTests.cs

```diff
@@ -117,22 +117,37 @@ public void Locate_FailedToReadCMakeSettings_NonCriticalException_Null()
         [TestMethod]
         public void Locate_FailedToParseCMakeSettings_Null()
         {
+            const string invalidJon = "invalid json";
+            var expectedMessage = GetExpectedDeserializationMessage();
```

Comment:
I didn't want to search for a hard-coded string, so the easiest thing to do seemed to be to generate the expected exception message by deserializing the same invalid json and catching the exception.